src/connectors/asana/mod.ts

```diff
@@ -0,0 +1,28 @@
+/** This file was autogenerated. Follow the steps in src/schema/slack/functions/README.md to rebuild **/
+import CreateComment from "./functions/create_comment.ts";
+import CreateProject from "./functions/create_project.ts";
+import CreateTask from "./functions/create_task.ts";
+import UpdateTask from "./functions/update_task.ts";
+
+const Asana = {
+  functions: {
+    /**
+     * @see The {@link https://api.slack.com/reference/connectors/asana/create_comment CreateComment} documentation.
```

Comment:
These URLs are not live yet, so maybe we shouldn't merge this until it is live?

Reply:
Lets wait until we are ready to release the new connector before merging

In the future wait for the docs in order to merge into `main` or wait wait for them before releasing a new version of the package?

Reply:
Good question! I guess it is OK to release if the docs aren't ready, it's just immediate adopters might suffer a poor experience (i.e. hitting a 404 if they try to load the docs URL)